serve.config.js

```diff
@@ -12,5 +12,5 @@ module.exports.serve = {
     publicPath: `/dist/`
   },
 
-  port: 9000
+  port: 9091
```

Comment:
Why was the port changed? If there’s no need for it, please leave it at 9000 which is the convention established in [Atlas Package](https://github.com/ebi-gene-expression-group/atlas-package).